Emby.AutoOrganize/Data/SqliteExtensions.cs

```diff
@@ -39,7 +39,7 @@ public static byte[] ToGuidBlob(this Guid guid)
 
         public static Guid ReadGuidFromBlob(this IResultSetValue result)
         {
-            return new Guid(result.ToBlob());
+            return new Guid(result.ToBlob().ToArray());
```

Comment:
This was the only build error caused by updating the SQLite version. Simple update to fix, the `ToBlob()` method now returns a `Span<>` that must be converted to an array.